app/api/endpoints/bfLoss/list/v3/ListBFLossesService.scala

```diff
@@ -16,30 +16,34 @@
 
 package api.endpoints.bfLoss.list.v3
 
+import api.controllers.RequestContext
 import api.endpoints.bfLoss.connector.v3.BFLossConnector
 import api.endpoints.bfLoss.list.v3.request.ListBFLossesRequest
 import api.models.errors._
-import api.services.DownstreamServiceSupport
 import api.services.v3.Outcomes.ListBFLossesOutcome
-import uk.gov.hmrc.http.HeaderCarrier
+import api.services.{ BaseService, DownstreamResponseMappingSupport }
+import utils.Logging
 
 import javax.inject.Inject
 import scala.concurrent.{ ExecutionContext, Future }
 
-class ListBFLossesService @Inject()(connector: BFLossConnector) extends DownstreamServiceSupport {
+class ListBFLossesService @Inject() (connector: BFLossConnector) extends BaseService with DownstreamResponseMappingSupport with Logging {
 
-  override val serviceName: String = this.getClass.getSimpleName
+  def listBFLosses(request: ListBFLossesRequest)(implicit ctx: RequestContext, ec: ExecutionContext): Future[ListBFLossesOutcome] =
+    connector
+      .listBFLosses(request)
+      .map {
+        case Left(err) =>
+          Left(mapDownstreamErrors(errorMap)(err))
 
-  def listBFLosses(request: ListBFLossesRequest)(implicit
-                                                 hc: HeaderCarrier,
-                                                 ec: ExecutionContext,
-                                                 correlationId: String): Future[ListBFLossesOutcome] = {
-    connector.listBFLosses(request).map {
-      mapToVendorDirect("listBFLosses", downstreamErrorMap)
-    }
-  }
+        case Right(responseWrapper) if responseWrapper.responseData.losses.isEmpty =>
+          Left(ErrorWrapper(ctx.correlationId, NotFoundError))
```

Comment:
This doesn't appear to be covered by a test (unit or integration). 

(Similarly for list loss claims - though that does have integration test.)